### PR TITLE
[Code Quality] Null pointer dereference check #528

### PR DESCRIPTION
**Pull request by harytary**
I ran the ctest and also infer in my local repo to ensure that issue is resolved
https://github.com/harytary/WasmEdge/actions/runs/1656862290

### CODE REVIEW

File: lib/executor/engine/variableInstr.cpp [hunk 1]
```diff
@@ -38,7 +38,9 @@ Expect<void> Executor::runGlobalGetOp(Runtime::StoreManager &StoreMgr,
 Expect<void> Executor::runGlobalSetOp(Runtime::StoreManager &StoreMgr,
                                       const uint32_t Idx) {
   auto *GlobInst = getGlobInstByIdx(StoreMgr, Idx);
-  GlobInst->getValue() = StackMgr.pop();
+  if (GlobInst) {
```
Comment (hydai) [inline lib/executor/engine/variableInstr.cpp:41]: Please use `assuming` to check the `nullptr`.

Ref: https://github.com/WasmEdge/WasmEdge/blob/bee4ad9831dec02d3224b9005d69cf30e3e973f6/lib/executor/instantiate/data.cpp#L38

**Review by hydai — CHANGES_REQUESTED** [review]

File: lib/executor/engine/variableInstr.cpp (hunk 1 above)
Comment (harytary) [reply lib/executor/engine/variableInstr.cpp:41]: okay sure, thanks for the reference.

**Comment by hydai** [comment]
Please use `rebase` to make all of your commits with DCO sign.

**Comment by harytary** [comment]
recheck

**Review by hydai — APPROVED** [review]

**Pull request merged**